### DISCUSSION TITLE
Issue opening Cyberdrop-dl

### DISCUSSION THREAD

**Discussion by GitGud117** [jbsparrow/CyberDropDownloader#173] [General]
Having scrape failures with both individual Bunkr files and folders. Is this because Bunkr has new extensions (.fi/.pk)?

**Comment by Mrimpossiblemk** [comment]
@jbsparrow PLEASE LOOK INTO THIS MATTER

**Reply by NTFSvolume** [reply]
> @jbsparrow PLEASE LOOK INTO THIS MATTER

This has been fixed since v5.6.54
